app/frontend/components/pages/accounts/accountsDashboard.tsx

```diff
@@ -127,7 +127,7 @@ const AccountsDashboard = () => {
             <button
               className="button secondary balance refresh"
               disabled={shouldDisableSendingButton(walletOperationStatusType)}
-              onClick={reloadWalletInfo}
+              onClick={() => reloadWalletInfo(true)}
```

Comment:
I think it would be more sensible to limit the debouncing to the reload button, as @PeterBenc pointed out, it could otherwise impact other parts of the app that need reloading, and managing that can get unnecessarily complex, or did you have some strong reason to implement it at "reloadWalletInfo" level?

If not, I guess something like this https://www.npmjs.com/package/debounce (let's port the code to our codebase if we don't have an equivalent) should work and the idea is that the onClick handler would have an effect only once every 30 seconds

Reply:
The implication is that every refresh button we have would have a separate debouncing though. An alternative could be to introduce a decorated "debouncedReloadWalletInfo" which would be a wrapper over reloadWalletInfo with debouncing added which would apply globally for all its invocations, and I think it would be cleaner than having a flag for that as it would be clear from the name what it's doing.